openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -1112,6 +1112,10 @@ def delete_library_block(usage_key, remove_from_parent=True):
 
     authoring_api.soft_delete_draft(component.pk)
 
+    # Import here to avoid breaking the LMS
+    from cms.djangoapps.contentstore import helpers as studio_helpers
+    studio_helpers.make_downstream_tags_editable(usage_key)
+
```

Comment:
We can handle inside the `LIBRARY_BLOCK_DELETED` signal handler (preferably to a celery task) and avoid cms import inside openedx/core. Maybe add the signal handler in `cms/djangoapps/contentstore/signals/handlers.py`.

Reply:
Ah good point -- see [4cba25c](https://github.com/openedx/edx-platform/pull/36228/commits/4cba25cd3f606f5770fcc7ea3a6a400e08d0491b) (also rebased against your updated PR).